guides/sdk-docs/public-apis/graphql-api.md

```diff
@@ -108,12 +93,15 @@ mutation MyMutation($eventId: String!) {
 ```
 
 ## 2. Generate React Hooks
+
 Call this command:
+
 ```sh
 nx run [feature-name]-[name-of-your-extension]:graphql-hooks
 ```
 
-## 3. Use it in your React code 
+## 3. Use it in your React code
+
```

Comment:
mutation under this is importing GetAssetsByProductionYearDocument it should be MyMutationDocument